docs/Project.toml

```diff
@@ -1,9 +1,16 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
+EnzymeCore = "f151be2c-9106-41f4-ab19-57ee4f262869"
+EnzymeTestUtils = "12d8515a-0907-448a-8884-5fe00fdf1c5a"
 Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Documenter = "1"
 Literate = "2"
+
+[sources]
```

Comment:
Is this in in 1.11?

Reply:
Yes: https://julialang.org/blog/2024/10/julia-1.11-highlights/#sources_section_in_projecttoml_in_pkgjl